File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.py
def solution(n):
    answer = 0
    if n % 7 == 0:
        answer = n // 7
    else:
        if n // 7 > 1:
            answer = n // 7 + 1
        elif 0 < n // 7 or n // 7 <= 1:
            answer = 1
    return answer

File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        if( n % 7 == 0){
            answer = n/7;
        }else{
            if(n/7 > 1){
                answer = n/7 + 1;
            }
            else if(0 < n/7 || n/7 <= 1){
                answer = 1;
            }
        }
        return answer;
    }
}